docs/TROUBLESHOOTING.md

```diff
@@ -85,6 +85,17 @@ Alternatively, the Docker event logs can be viewed using SSH with the following
   PS C:\Users\username> Get-EventLog -LogName Application -Source Docker
   ```
 
```

Comment:
Can you add the following here -
`Requires SSH [bastion](https://github.com/eparis/ssh-bastion) deployed on cluster`